README.md

```diff
@@ -84,35 +84,35 @@ defined('YII_ENV') or define('YII_ENV', 'dev');
 // Alternatives set this in your rr.yaml file
 //defined('YII_ENV') or define('YII_ENV', \getenv('YII_ENV'));
 
-// Load your vendor dependencies and Yii2 autoloader
-require_once '/path/to/vendor/autoload.php';
-require_once '/path/to/vendor/yiisoft/yii2/Yii.php';
+require __DIR__ . '/../vendor/autoload.php';
+require __DIR__ . '/../vendor/yiisoft/yii2/Yii.php';
 
-// Roadrunner relay and PSR7 object
-$relay = new \Spiral\Goridge\StreamRelay(STDIN, STDOUT);
-$psr7 = new \Spiral\RoadRunner\PSR7Client(new \Spiral\RoadRunner\Worker($relay));
+$worker = Spiral\RoadRunner\Worker::create();
+$psrFactory = new Nyholm\Psr7\Factory\Psr17Factory();
```

Comment:
I'd prefer there to be a generic PSR17 implementation, as opposed to a specific one. My concern with this one specifically is that it proposes specifying a default HTTP response code, when in reality the bridge itself can handle it's own response codes.

Reply:
Sure, I will use Laminas so we don't need to import anything else.